### PR TITLE
U402/t436 send relevant info

### PR DESCRIPTION
**Pull request by Discojanne**

### CODE REVIEW

**Comment by Discojanne** [comment]
testa gärna multiplayer

File: Sail/src/Sail/entities/systems/network/receivers/NetworkReceiverSystemHost.cpp [hunk 1]
```diff
@@ -16,4 +21,73 @@ void NetworkReceiverSystemHost::handleIncomingData(std::string data) {
...
+void NetworkReceiverSystemHost::endMatch() {
+
+	m_startEndGameTimer = true;
+}
+
+void NetworkReceiverSystemHost::endMatchAfterTimer() {
+
+	static float endGameClock;
+	if (m_startEndGameTimer) {
+		endGameClock++;
```
Comment (whalemane) [inline Sail/src/Sail/entities/systems/network/receivers/NetworkReceiverSystemHost.cpp]: This should probably use dt, so it is the same for everyone

File: Sail/src/Sail/entities/systems/network/receivers/NetworkReceiverSystem.cpp [hunk 2]
```diff
@@ -286,6 +295,10 @@ void NetworkReceiverSystem::update() {
 				int nKills;
 				int placement;
 
+				int bulletsFired, jumpsMade;
+				float distanceWalked;
+				Netcode::PlayerID bfID, dwID, jmID;
```
Comment (whalemane) [inline Sail/src/Sail/entities/systems/network/receivers/NetworkReceiverSystem.cpp]: these variable names are pretty vague, could be more descriptive

File: Sail/src/Sail/entities/systems/network/receivers/NetworkReceiverSystemHost.cpp [hunk 3]
```diff
@@ -16,4 +21,73 @@ void NetworkReceiverSystemHost::handleIncomingData(std::string data) {
...
+		gts->bfID = id;
+	}
+	if (dw > gts->distanceWalked) {
+		gts->distanceWalked = dw;
+		gts->dwID = id;
+	}
+	if (jm > gts->jumpsMade) {
+		gts->jumpsMade = jm;
+		gts->jmID = id;
+	}
```
Comment (whalemane) [inline Sail/src/Sail/entities/systems/network/receivers/NetworkReceiverSystemHost.cpp]: These variables are a bit vague as well

**Pull request merged**